x-pack/plugin/transform/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/continuous/DateHistogramGroupByIT.java

```diff
@@ -43,15 +43,21 @@
         .format(Instant.ofEpochMilli(42));
 
     private final boolean missing;
+    private final boolean dateAsEpochMillis;
 
     public DateHistogramGroupByIT() {
         missing = randomBoolean();
+        dateAsEpochMillis = randomBoolean();
```

Comment:
Why is this the only test that does it? Wouldn't it be applicable for example for `DateHistogramGroupByOtherTimeFieldIT.java` as well?